manifests/tls-issuers.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 apiVersion: cert-manager.io/v1
-kind: ClusterIssuer
+kind: Issuer
```

Comment:
Issuer's are namespace scoped, so you will need to specify a namespace for convenience and add it to the values so it can be configure